packages/aws-cdk/test/integ/cli/aws-helpers.ts

```diff
@@ -5,12 +5,72 @@ interface Env {
   region: string;
 }
 
+function chainableCredentials(region: string): AWS.Credentials | undefined {
+
+  const profileName = process.env.AWS_PROFILE;
+  if (process.env.CODEBUILD_BUILD_ARN && profileName) {
+
+    // in codebuild we must assume the role that the cdk uses
+    // otherwise credentials will just be picked up by the normal sdk
+    // heuristics and expire after an hour.
+
+    // can't use '~' since the SDK doesn't seem to expand it...?
```

Comment:
Expanding `~` is a bash feature, not any of the underlying OS or libraries.

Reply:
Sure, but can easily be implemented and simulated by them...

```python
import os

os.path.expanduser('~/.aws') # /Users/epolon/.aws
```

Reply:
Python has *everything* so that's not really a fair comparison :)